conf/cluster-master/application.conf

```diff
@@ -69,3 +70,20 @@ task.status.reportintervalinms=1000
 # Enable metrics / events
 metrics.enabled=true
 
+# UI
+admin.server.enabled=true
```

Comment:
so that first time user dont have to figure this out, if there is any specific reason not to turn it on by default, then i can remove this config. I have added those configs that I had to add when i setup the gobblin first time to get it working.

Reply:
@ibuenros, made default to false as before, since it requires store config as well.